.editorconfig

```diff
@@ -1,11 +1,11 @@
 root = true
 
 [*]
-indent_style = tab
-indent_size = 2
 charset = utf-8
-trim_trailing_whitespace = true
-insert_final_newline = true
 end_of_line = lf
-max_line_length = null
+indent_size = 2
+indent_style = tab
 insert_final_newline = true
+insert_final_newline = true
+max_line_length = 120
```

Comment:
Why is it not 80? Default is 80 and should stay like this.

Reply:
@igeligel I'm fine with 80, but @ostoc likes longer lines. Given that he's the main author of this repo, I'd consider that to be more important than following an arbitrary standard.

I personally had no issues reading it.

Reply:
> arbitrary standard

This is the community standard. If you line-length reach more than 80 characters, then the code should be refactored since it is not clear anymore and easy to read.

Reply:
Because this is what prettier has in mind 👍 

Read through their option philosophy here: https://prettier.io/docs/en/option-philosophy.html

I think with our configuration:

```json
{
  "useTabs": true,
  "singleQuote": true,
  "trailingComma": "all",
  "arrowParens": "always"
}
```

We do not have that many changes to the default prettier configuration.